### PR TITLE
rad binaries

### PR DESCRIPTION
**Pull request by FintanH**
Since we now have a `bins` section, I'm adding the `rad`, `rad-identities-dev`, and `rad-profile-dev` binaries there. Note that I'm building this on top of `finto/rad-identities`.

### CODE REVIEW

**Comment by kim** [comment]
Out of curiosity: is there a deeper reason to prefer crate-per-binary over just one crate with multiple bin targets?

**Comment by FintanH** [comment]
> Out of curiosity: is there a deeper reason to prefer crate-per-binary over just one crate with multiple bin targets?

My thinking was that dependencies are separated and we continue on the small, well-defined crates path. What do you think?

File: bins/rad-identities-dev/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rad-identities-dev"
+version = "0.1.0"
+authors = ["Fintan Halpenny fintan.halpenny@gmail.com"]
```
Comment (kim) [inline bins/rad-identities-dev/Cargo.toml:4]: does cargo not want some angle brackets around the email address?

Comment (FintanH) [reply bins/rad-identities-dev/Cargo.toml:4]: Yes, apparently[0]

[0]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field

**Comment by kim** [comment]
> My thinking was that dependencies are separated and we continue on the
> small, well-defined crates path. What do you think?

I think that it will most likely become a bit of a versioning nightmare once we
publish actual cargo versions. But we can revisit once we get there.

File: .buildkite/pipeline.yaml [hunk 2]
```diff
@@ -69,3 +69,16 @@ steps:
       - "Cargo.lock"
     agents: *build-agent
     env: *build-docker
+
+  - label: "Build Bins"
```
Comment (xla) [inline .buildkite/pipeline.yaml:73]: Mea culpa!

File: bins/rad-identities-dev/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+[dependencies]
+anyhow = "1"
+structopt = "0.3"
+
+[dependencies.rad-exe]
+path = "../../rad-exe"
+
+[dependencies.rad-identities]
+path = "../../rad-identities"
+
```
Comment (xla) [inline bins/rad-identities-dev/Cargo.toml:17]: Generous newline?

Comment (FintanH) [reply bins/rad-identities-dev/Cargo.toml:17]: Was feeling nice

**Review by xla — APPROVED** [review]
🛋 🔣 🍡 🌟

**Pull request merged**